GitHub.Collectors/Model/DataContract.cs

```diff
@@ -5,6 +5,8 @@ namespace Microsoft.CloudMine.GitHub.Collectors.Model
 {
     public class DataContract
     {
+        public const string OrganizationInstanceRecordType = "GitHub.OrgIstance";
```

Comment:
>OrgIstance [](http://example.com/codeflow?start=69&length=10)

Can you cross-check with Sascha to see how painful it would be to fix this e.g., typo here (Instance instead of Istance)?

Reply:
Discussed with Sascha, we are not doing this yet. Please add a comment that at some point this needs to be fixed.